alicloud/resource_alicloud_fc_trigger.go

```diff
@@ -71,6 +71,7 @@ func resourceAlicloudFCTrigger() *schema.Resource {
 			"config": {
 				Type:     schema.TypeString,
 				Required: true,
+				ForceNew: true,
```

Comment:
`config` can be modified and it should not be `ForceNew: true`.

Reply:
This is mandatory when `type = "mns_topic"` because their API (and even the web console) don't support update. If I don't add this feature, `terraform apply` with a modified trigger configuration will simply fail and the user will be stuck.